crates/lib3h/src/dht/mirror_dht.rs

```diff
@@ -326,44 +335,43 @@ impl MirrorDht {
                     MirrorGossip::Entry(entry) => {
                         let diff = self.diff_aspects(&entry);
                         if diff.len() > 0 {
-                            return Ok(vec![DhtEvent::HoldEntryRequested(
-                                self.this_peer.peer_address.clone(),
-                                entry,
-                            )]);
+                            self.endpoint_self
+                                .publish(DhtRequestToParent::HoldEntryRequested {
+                                    from_peer: self.this_peer.peer_address.clone(),
+                                    entry,
+                                });
                         }
-                        return Ok(vec![]);
                     }
                     MirrorGossip::Peer(gossiped_peer) => {
                         let maybe_known_peer = self.get_peer(&gossiped_peer.peer_address);
-                        if maybe_known_peer.is_none() {
-                            return Ok(vec![DhtEvent::HoldPeerRequested(gossiped_peer)]);
+                        match maybe_known_peer {
+                            None => {
+                                self.endpoint_self
+                                    .publish(DhtRequestToParent::HoldPeerRequested(
+                                        gossiped_peer.clone(),
+                                    ));
+                            }
+                            Some(known_peer) => {
+                                // Update Peer timestamp
+                                if gossiped_peer.timestamp > known_peer.timestamp {
+                                    let _ = self.add_peer(&gossiped_peer);
```

Comment:
It's not obvious why the return value to `add_peer` is ignored here and in a few other places. maybe just give the unused variable it a meaningful name or document it.

Reply:
The return value tells us if the added content is new or just an update.
Here we already know its known content so we don't care about this return value.